src/main/java/com/arcanewarrior/disguises/Disguise.java

```diff
@@ -14,6 +15,11 @@ public Disguise(@NotNull EntityType entityType, @NotNull Player player, @NotNull
         super(entityType);
         this.player = player;
         this.manager = manager;
+        if(MinestomDisguises.getInstance().getConfiguration().getTable("disguises").getBoolean("translate-player-names", true)) {
```

Comment:
This is not really that nice, it should be a config that's held onto by disguise manager, not accessed statically